### PR TITLE
create OCaml team and add to team-list

### PR DESCRIPTION
**Pull request by Alizter**
- Created an OCaml entry in the team-list file.
- Add @NixOS/ocaml as CODEOWNERS for OCaml ecosystem

@romildo @superherointj @ulrikstrid should I add you to the team-list entry?

Also @vbgl would you like to join the OCaml team and team-list?

In theory, if I have setup the CODEOWNERS file correctly, we can get a notification everytime the mentioned files get edited.

The CODEOWNERS file seems to give an error indicating that NixOS/ocaml doesn't have write permission to the repository. What does this mean for teams? Edit: Ah I see that every member of the team needs to have write permission for this to work. Not sure what to do about that.

### CODE REVIEW

**Comment by superherointj** [comment]
Members of teams receive commiter rights automatically.
It is not possible to use this functionality as you expect it to because you are not a commiter yet.
I suggest you drop this idea and just add your username to CODEOWNERS so you get notifications.
Also, feel free to add the ocaml file at top-level which is missing.

**Comment by Alizter** [comment]
@superherointj OK, I've updated it now. Does that look OK?

**Comment by superherointj** [comment]
> @superherointj OK, I've updated it now. Does that look OK?

It seems you need commiter rights first. At least, it is the error message that shows for me.

Also, you should have removed teams. Don't know if that is affecting it.

File: maintainers/team-list.nix [hunk 1]
```diff
@@ -568,6 +568,14 @@ with lib.maintainers; {
...
   };
 
+  ocaml = {
+    members = [
+      alizter
+    ];
+    scope = "Maintain the OCaml ecosystem.";
+    shortName = "OCaml";
+  };
+
```
Comment (superherointj) [inline maintainers/team-list.nix:578]:
```suggestion
```

**Comment by Alizter** [comment]
I don't think that is affecting it. The team-list isn't related to the GitHub teams, from what I have read it just allows a bot to ping people during a feature freeze. I essentially modelled it off of the Coq team and it serves as documentation about who maintains what.

**Comment by Alizter** [comment]
I guess I can mark this as draft for the time being while I ask for commit permission.

**Comment by superherointj** [comment]
> I don't think that is affecting it. The team-list isn't related to the GitHub teams, from what I have read it just allows a bot to ping people during a feature freeze. I essentially modelled it off of the Coq team and it serves as documentation about who maintains what.

It shows a message error for me.

> I guess I can mark this as draft for the time being while I ask for commit permission.

Please close it. Whenever you have commiter rights reopen it. There is already too many PRs.

**Pull request closed without merging**